core/src/components/button/button.ionic.scss

```
@@ -102,44 +101,24 @@
 // Button Sizes
 // -------------------------------------------------------------------------------
 
-/* Extra Small Button */
-:host(.button-xsmall) {
-  --padding-top: #{globals.$ion-space-100};
-  --padding-end: #{globals.$ion-space-300};
-
-  min-height: globals.$ion-space-600;
-
-  font-size: globals.$ion-font-size-300;
-}
-
 /* Small Button */
 :host(.button-small) {
   --padding-top: #{globals.$ion-space-200};
   --padding-end: #{globals.$ion-space-400};
```

```suggestion
  --padding-end: #{globals.$ion-space-400};
```

`--padding-top` can be removed because it should be `0` which is defined in `:host`